cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -66,8 +57,10 @@ pub enum YaccGrammarErrorKind {
 /// Any error from the Yacc parser returns an instance of this struct.
 #[derive(Debug, PartialEq, Eq)]
 pub struct YaccGrammarError {
-    pub kind: YaccGrammarErrorKind,
-    pub span: Span,
+    pub(crate) kind: YaccGrammarErrorKind,
+    // contains at least 1 span.
```

Comment:
These need to be (nicely formatted :)) `///` doc strings.

Reply:
Should be fixed in https://github.com/softdevteam/grmtools/pull/322/commits/65d12bfc088d9a8d69cf7824da79daa22424666b